### ISSUE TITLE
[Feature] Move a file to a separate window

### ISSUE THREAD

**Issue by adiee5** [WerWolv/ImHex#1854]
### What feature would you like to see?

As of now, you either have files tabbed in one window or you have to manually execute a new instance of ImHex to have a file in a separate window from the others. This FR suggests, that user could drag out a file tab from its placement in order to spawn a new ImHex window, just like it's done on most web browsers. Alternatively, give the user ability to have multiple file views in one window like in vscode, if that's better to implement.

### How will this feature be useful to you and others?

This can make it easier to visually compare multiple files, as having to manually run a new ImHex instance to see more files at once is not really comfortable

### Request Type

- [ ] I can provide a PoC for this feature or am willing to work on it myself and submit a PR

**Comment by adiee5** [comment]
Yeah, it seems like nobody noticed this  feature request yet. I'll try to check whether the feature was already added in some recent version

**Comment by WerWolv** [comment]
As of right now, it's not possible yet and it would probably require quite a lot of changes to get it working. Many things currently work under the assumption that there's only ever one file visible at once

**Comment by adiee5** [comment]
> Many things currently work under the assumption that there's only ever one file visible at once

My idea was to make it run separate imhex instance, as most text editors (and alike) do.

Still, obviously I'm aware, that this won't be easy task to accomplish (lots of inter-process communication), but it's probably still at least worth having this in a long term TODO list or something. But yeah, no hard pressure, this is amazing software ❤️‍🔥
